terraform/github/data-catalogue.tf

```diff
@@ -33,6 +33,23 @@ module "data_catalogue_repository" {
   }
 }
 
+module "datahub_custom_api_source" {
+  source = "./modules/repository"
+
+  name        = "data-catalogue"
```

Comment:
This needs updating to the name you want to use, and then you'll need to update the module name to be `new_name_repository`

Reply:
🤦🏻  ah of course